client/app/directives/ngRightClick.js

```diff
@@ -0,0 +1,22 @@
+(function (angular) {
+    "use strict";
+
+    var module = angular.module('myCloudDriveApp');
+	module.directive('ngRightClick', function ($document, $parse) {
+        return {
```

Comment:
I think it is not a good idea to name directives in such a way. It may conflict with Angular's directives.